chia/daemon/keychain_server.py

```diff
@@ -26,6 +32,33 @@
 KEYCHAIN_ERR_MALFORMED_REQUEST = "malformed request"
 
 
+@streamable
+@dataclass(frozen=True)
+class GetKeyRequest(Streamable):
+    fingerprint: uint32
+    include_secrets: bool = False
+
+
+@streamable
+@dataclass(frozen=True)
+class GetKeysRequest(Streamable):
+    include_secrets: bool = False
+
+
+def get_key_command(keychain: Keychain, request: GetKeyRequest) -> Dict[str, Any]:
+    return {
+        "success": True,
+        "key": keychain.get_key(request.fingerprint, request.include_secrets).to_json_dict(),
+    }
+
+
+def get_keys_command(keychain: Keychain, request: GetKeysRequest) -> Dict[str, Any]:
+    return {
```

Comment:
I like the compactness of the new RPC implementations, though it's hard to figure out the shape of the response data beyond the top-level keys. A comment with an example output might be sufficient, or maybe we should have a GetKeysResponse(Streamable) class?

Reply:
> or maybe we should have a GetKeysResponse(Streamable) class?

Yeah good idea, i changed it and also slightly modified the request class (it contains the `run` now which was a separate function before and passed into `run_command`.